src/mainviews/job-detail.tsx

```diff
@@ -109,203 +94,172 @@ export function JobDetail(props: IJobDetailProps): JSX.Element {
-                <TextFieldStyled
-                  label={trans.__('Idempotency token')}
-                  defaultValue={job?.idempotency_token ?? ''}
-                />
-                <FormLabel component="legend">{trans.__('Tags')}</FormLabel>
-                {job?.tags &&
-                  job?.tags.map(tag => (
-                    <TextFieldStyled
-                      label={trans.__('Tag')}
-                      defaultValue={tag}
-                      InputProps={{
-                        readOnly: true
-                      }}
-                    />
-                  ))}
-                <TextFieldStyled
-                  label={trans.__('Create time')}
-                  defaultValue={timestampLocalize(job?.create_time ?? '')}
-                />
+  const ButtonBar = () => (
+    <Stack direction="row" spacing={1} justifyContent="flex-end">
+      {job?.status === 'IN_PROGRESS' && (
+        <Button variant="outlined" onClick={handleStopJob}>
+          {trans.__('Stop Job')}
+        </Button>
+      )}
+      <Button variant="outlined" onClick={rerunJob}>
+        {trans.__('Rerun Job')}
+      </Button>
+      <Button variant="contained" color="error" onClick={handleDeleteJob}>
+        {trans.__('Delete Job')}
+      </Button>
+    </Stack>
+  );
+
+  const coreOptionsFields: TextFieldProps[] = [
+    { defaultValue: job?.name ?? '', label: trans.__('Job name') },
+    { defaultValue: job?.job_id ?? '', label: trans.__('Job ID') },
+    {
+      defaultValue: job?.input_uri ?? '',
+      label: trans.__('Input file')
+    },
+    {
+      defaultValue: job?.output_uri ?? '',
+      label: trans.__('Output path')
+    },
+    {
+      defaultValue: job?.runtime_environment_name ?? '',
+      label: trans.__('Environment')
+    },
+    {
+      defaultValue: timestampLocalize(job?.create_time ?? ''),
+      label: trans.__('Create time')
+    },
+    {
+      defaultValue: timestampLocalize(job?.start_time ?? ''),
+      label: trans.__('Start time')
+    },
+    {
+      defaultValue: timestampLocalize(job?.end_time ?? ''),
+      label: trans.__('End time')
+    },
+    { defaultValue: job?.status ?? '', label: trans.__('Status') }
+  ];
+
+  const CoreOptions = () => (
+    <Card>
+      <CardContent>
+        <Grid container spacing={4}>
+          {coreOptionsFields.map(textFieldProps => (
+            <Grid key={textFieldProps.defaultValue as string} xs={12} md={6}>
+              <TextFieldStyled {...textFieldProps} />
+            </Grid>
+          ))}
+        </Grid>
+      </CardContent>
+    </Card>
+  );
+
+  const Parameters = () => (
+    <Card>
+      <CardContent>
+        <FormLabel sx={{ mb: 4 }} component="legend">
+          {trans.__('Parameters')}
+        </FormLabel>
+        <Grid container spacing={4}>
+          {Object.entries(job?.parameters ?? {}).map(([parameter, value]) => (
+            <React.Fragment key={parameter}>
+              <Grid xs={12} md={6}>
```

Comment:
The docs recommend using `<Grid item` inside a `<Grid container`.

Reply:
It is correct for [Grid](https://mui.com/material-ui/react-grid/). [Grid 2 ](https://mui.com/material-ui/react-grid2/)(or `@mui/material/Unstable_Grid2`) gets rid of item prop and some other deficiencies. I decided on using grid based on the fact that it gives clearer code and and that mui website lists it as 'new implementation'. I do agree that it might be better to use stable Grid version for the sake of risk aversion:
![Screen Shot 2022-09-29 at 9 52 25 AM](https://user-images.githubusercontent.com/26686070/193091712-d2e3dbe0-aafa-4022-90ea-0d398de15510.png)